frontend/src/app/pages/exercises/exercise/shared/exercise-map/utility/popup-helper.ts

```diff
@@ -102,15 +106,22 @@ export class ImagePopupHelper {
     }
 
     /**
+     * @param layer The layer the feature is in
      * @returns wether the feature is visible at the given pixel
      */
     private hasFeatureAtPixel(feature: Feature<Point>, pixel: Pixel): boolean {
-        return this.olMap
-            .getFeaturesAtPixel(pixel, {
-                // We assume the feature has no holes and is on the two axis the highest
-                hitTolerance: 0,
-            })
-            .some((_feature) => _feature.getId() === feature.getId());
+        return (
+            this.olMap.forEachFeatureAtPixel(
```

Comment:
From the documentation of this function:
````
@return
Callback result, i.e. the return value of last callback execution, or the first truthy callback return value.
````

Reply:
Maybe add this as a short comment?

Reply:
I thought about it, but it should be enough to have this documented on the `forEachFeatureAtPixel` function in ol (as a jsDoc comment -> hovering over it is enough).
If we duplicate this documentation, this could get out of sync.

I added the comment in GitHub only because you can't see the documentation by hovering over the function in GitHub.